app/assets/javascripts/discourse-common/lib/deprecated.js.es6

```diff
@@ -1,3 +1,8 @@
-export default function deprecated(msg) {
-  console.warn(`DEPRECATION: ${msg}`); // eslint-disable-line no-console
+export default function deprecated(msg, opts = {}) {
+  let warning = `DEPRECATION: ${msg}`;
+  let versionInfo = '';
+  if (opts.since) { versionInfo += `It's deprecated since Discourse ${opts.since}.`; }
+  if (opts.dropFrom) { versionInfo += `It will be removed from Discourse ${opts.dropFrom}.`; }
+  if (versionInfo.length > 0) { versionInfo = `\nVersion: ${versionInfo}\n`; }
+  console.warn(`DEPRECATION: ${warning}${versionInfo}`); // eslint-disable-line no-console
```

Comment:
I think you have `DEPRECATION:` twice here. (line 2 & line 7)

Reply:
Fixed.